<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TriggerUiClass</class>
 <widget class="QWidget" name="TriggerUiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>440</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Trigger</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{font-family:&quot;Microsoft YaHei&quot;;font-size:14px;}
#TriggerUiClass{background-color:#CEF;}

QCheckBox::indicator{background-color:#FFF;width:20px;height:20px;}
QCheckBox::indicator:checked{background-color:#888;}

QComboBox{background-color:white;padding-left:3px;border:none;font-size:12px;}
QComboBox::drop-down{background-color:#FFF;}
QComboBox QAbstractItemView{background-color:white;border:none;}

QLineEdit{background-color:white;border:none;font-size:12px;}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QLabel" name="lbTrigger">
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family:&quot;Microsoft YaHei&quot;;font-size:16px;</string>
     </property>
     <property name="text">
      <string>在此处设置宏的触发方式及按键</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="box">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableWidget" name="tbActions">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>460</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableWidget{
background-color:white;
border:none;
outline:none;
font-size:12px;
}
QTableWidget::item:hover{
background-color:#D8F8FF;
border:none;
}
QTableWidget::item::selected{
background-color:#CEF;
color:#000;
border:none;
}
QHeaderView{
background-color:white;
border:none;
}
QHeaderView::section{
background-color:white;
border:none;
}
QHeaderView::section:hover{
background-color:#C0E2F2;
border:none;
}</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="tabKeyNavigation">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideRight</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="enBox">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="blockBox">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QCheckBox" name="chbBlock">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>屏蔽触发键</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="modeBox">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="lbMode">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>触发模式</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbMode">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="toolTip">
            <string>切换：点击触发键执行宏，再次点击触发键停止
按下：按下触发键执行宏，直到完成对应次数或松开
松开：松开触发键执行宏，直到完成对应次数或按下</string>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="keyBox">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="lbTr">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>触发按键</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QKeyEdit" name="hkTr" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>24</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:white;font-size:12px;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="countBox">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="lbCount">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>触发次数</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="etCount">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="statusTip">
            <string/>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="dragEnabled">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>0~9999（0 = 无限）</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QKeyEdit</class>
   <extends>QWidget</extends>
   <header>QKeyEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
